sunkit_pyvista/tests/test_pyvista.py

```diff
@@ -112,7 +112,7 @@ def test_clip_interval(aia171_test_map, plotter):
     plotter.plot_map(aia171_test_map, clip_interval=(1, 99)*u.percent)
     clim = plotter._get_clim(data=plotter.plotter.mesh['data'],
                              clip_interval=(1, 99)*u.percent)
-    expected_clim = [0.006716044038535769, 0.8024368512284383]
+    expected_clim = [0.15782961272423796, 0.8477792344775393]
```

Comment:
This change is because the clipping is now done on only the on-disk data, whereas previously it was done on the whole data array.